db/migrate/20230601112439_create_orders.rb

```diff
@@ -0,0 +1,16 @@
+class CreateOrders < ActiveRecord::Migration[7.0]
+  def change
+    create_table :orders do |t|
+      t.integer :status, default: 0
+      t.references :user, null: false, foreign_key: true
+      t.references :branch, null: false, foreign_key: true
+      t.integer :total, default: 0
```

Comment:
This should be decimal type as we can have total in points.

Reply:
Done